README.md

```diff
@@ -1,7 +1,7 @@
 [![Logo](/logo.png)](https://30secondsofcode.org/js/p/1)
 
 # 30 seconds of code
-
+## Mahmood
```

Comment:
Spam on this repository won't be tolerated. Next time, you'll be blocked permanently. Please be respectful of people's time. Have a nice day!